-- Lower setuptools version requirement to be compatible with the version
-- available in FreeBSD ports (63.x).

--- pyproject.toml.orig	2026-10-10 20:42:00 UTC
+++ pyproject.toml
@@ -107,7 +107,7 @@ requires = [
 
 [build-system]
 requires = [
-  "setuptools>=64", 
+  "setuptools>=0", 
   "setuptools_scm[toml]>=8"
 ]
 build-backend = "setuptools.build_meta"
